simplesat/compute_dependencies.py

```diff
@@ -63,6 +63,42 @@ def compute_reverse_dependencies(repositories, requirement, transitive=False):
     return dependencies
 
 
+def compute_leaf_packages(repositories):
+    """ Compute the leaf packages in `repositories`. Leaf packages are packages
+    with no reverse dependencies.
+
+    Parameters
+    ----------------
+    repositories : iterable of Repository objects
+
+    Returns
+    -----------
+    dependencies : set
+        Set of leaf packages in the given repositories.
+    """
+    pool = Pool(repositories)
```

Comment:
This first bit is shared with `compute_reverse_dependencies`--they should use a common utility